### DISCUSSION TITLE
Cannot import project from github - TransformStream is not defined

### DISCUSSION THREAD

**Discussion by trompx** [vercel/community#5020] [Help]
### Question

When I click on the **import** button of the project I wish to deploy on Vercel, I'm redirected to a page with the message:
`TransformStream is not defined`
So I can't deploy it, what should I do please?

**Comment by trompx** [comment]
FYI, this must be a bug, it happens only on Firefox, working on chrome.

**Reply by amyegan** [reply]
What version of Firefox are you using?
